<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AdsPower API接口地址:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leURL">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>http://local.adspower.com:50325</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>连接 / 刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号列表：</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="twBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>611</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>730</y>
     <width>701</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>AdsPower接口1秒只能访问1次，刷新列表时请等待
点击 连接/刷新 时，会自动打开账号列表中第一个账号窗口以获得Chrome驱动的位置</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开  始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>停  止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清  屏</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teDisplay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>441</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>窗口位置： X                px      Y                px</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbX">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-4096</number>
   </property>
   <property name="maximum">
    <number>4096</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>-8</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbY">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-2160</number>
   </property>
   <property name="maximum">
    <number>2160</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>-8</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbHeight">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2160</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>1040</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>窗口尺寸： 宽度                px      高度               px</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbWidth">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4096</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>960</number>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>leURL</zorder>
  <zorder>btnConnect</zorder>
  <zorder>label</zorder>
  <zorder>twBrowser</zorder>
  <zorder>label_3</zorder>
  <zorder>btnStart</zorder>
  <zorder>btnStop</zorder>
  <zorder>btnClear</zorder>
  <zorder>teDisplay</zorder>
  <zorder>label_4</zorder>
  <zorder>sbX</zorder>
  <zorder>sbY</zorder>
  <zorder>label_5</zorder>
  <zorder>sbWidth</zorder>
  <zorder>sbHeight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
